enclave-tls/src/crypto_wrappers/api/crypto_wrapper_register.c

```diff
@@ -16,6 +17,14 @@ crypto_wrapper_err_t crypto_wrapper_register(const crypto_wrapper_opts_t *opts)
 
 	ETLS_DEBUG("registering the crypto wrapper '%s' ...\n", opts->type);
 
+	if (opts->flags & CRYPTO_WRAPPER_OPTS_FLAGS_SGX_ENCLAVE) {
+		if (!is_sgx_supported_and_configured()) {
+			ETLS_ERR("failed to register the crypto wrapper '%s' \
```

Comment:
please remove \, otherwise the table character will be filled into the string as below:
failed to register the crypto wrapper 'xx'                      due to lack of SGX capability